packages/js/product-editor/src/components/block-editor/style.scss

```diff
@@ -15,4 +17,52 @@
 		margin-left: auto;
 		margin-right: auto;
 	}
+
+	.components-input-control {
```

Comment:
Could we update the placeholder colors to `#757575` (gray 700)?

Reply:
I think labels should be `#1e1e1e` as well.  (gray 900)

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/37805/commits/aed6675ea178c3df423999f2e4d139670661d12f